src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbs.tsx

```diff
@@ -54,32 +50,51 @@ type Props = {
 
 class Breadcrumbs extends React.Component<Props, State> {
   state: State = {
-    isCollapsed: true,
     searchTerm: '',
     breadcrumbs: [],
     filteredByFilter: [],
-    filteredByCustomSearch: [],
-    filteredBreadcrumbs: [],
+    filteredBySearch: [],
     filterOptions: [[], []],
   };
 
   componentDidMount() {
     this.loadBreadcrumbs();
   }
 
-  componentDidUpdate() {
-    this.loadListBodyHeight();
+  componentDidUpdate(_prevProps: Props, prevState: State) {
+    if (
+      prevState.breadcrumbs.length === 0 &&
+      this.state.breadcrumbs.length >= MAX_CRUMBS_WHEN_COLLAPSED
+    ) {
+      this.expandCollapsedCrumbs();
+    }
   }
 
   listBodyRef = React.createRef<HTMLDivElement>();
 
-  loadListBodyHeight = () => {
-    if (!this.state.listBodyHeight) {
-      const offsetHeight = this.listBodyRef?.current?.offsetHeight;
-      this.setState({
-        listBodyHeight: offsetHeight ? `${offsetHeight}px` : 'none',
-      });
+  expandCollapsedCrumbs = () => {
+    this.setState(
+      prevState => ({
+        filteredBySearch: prevState.breadcrumbs,
+      }),
+      () => {
+        this.scrollToTheBottom();
+      }
+    );
+  };
+
+  scrollToTheBottom = () => {
+    const element = this.listBodyRef?.current;
+
+    if (!element) {
+      return;
     }
+
+    element.scrollTo({
+      top: element.scrollHeight,
+      left: 0,
+      behavior: 'smooth',
```

Comment:
```suggestion
```

Can we get rid of this? I don't see the use of animating the scroll it just takes longer and looks odd.